com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/publisher/Publisher.java

```diff
@@ -49,6 +46,8 @@ public String getSupportedVersions() {
         return this.settings.getSupportedVersions();
     }
 
+    public boolean ifShouldBeMovedIntoBundleFolder() { return this.settings.getSaveZipInBundleFolder(); }
```

Comment:
This will look weird with a double "if":

```
if (ifShouldBeMovedIntoBundleFolder()) {
}
```

I suggest to name it only `shouldBeMovedIntoBundleFolder`

Reply:
fixed
thx!